mesa/extra.py

```diff
@@ -8,10 +8,11 @@
 import Person
 import numpy as np
 import random
-
+from numpy.random import default_rng
+#The random generator used in all modules BitGenerator(PCG64)
+rng = default_rng(42) #If None, unpredictable entropy will be pulled from the OS.
```

Comment:
The default must be none.

Reply:
Correct, I put the comment for completeness.